tests/unit/conftest.py

```diff
@@ -170,6 +170,12 @@ def main_acoustic_fields(dbsession):
     return {r.field for r in records}
 
 
+@pytest.fixture
```

Comment:
Would it be useful to scope this fixture to the `session` level?

Reply:
```
ScopeMismatch: You tried to access the 'function' scoped fixture 'dbsession' with a 'session' scoped request object, involved factories
tests/unit/conftest.py:166:  def main_acoustic_fields(dbsession)
tests/unit/conftest.py:108:  def dbsession(connection)
```